docs/Configuration.md

```diff
@@ -31,31 +33,33 @@ for bp in browser_params:
 validate_manager_params(manager_params)
 validate_crawl_configs(manager_params, browser_params)
 ```
-**NOTE**: If any validations fail, we raise `ConfigError`
 
+**NOTE**: If any validations fail, we raise `ConfigError`
 
-* [Platform Configuration Options](#platform-configuration-options)
-* [Browser Configuration Options](#browser-configuration-options)
-* [Instruments](#instruments)
-  * [`http_instrument`](#http_instrument)
-  * [`js_instrument`](#js_instrument)
-  * [`navigation_instrument`](#navigation_instrument)
-  * [`callstack_instrument`](#callstack_instrument)
-  * [`dns_instrument`](#dns_instrument)
-  * [`cookie_instrument`](#cookie_instrument)
-* [Browser Profile Support](#browser-profile-support)
-  * [Stateful vs Stateless crawls](#stateful-vs-stateless-crawls)
-  * [Loading and saving a browser profile](#loading-and-saving-a-browser-profile)
-    * [Save a profile](#save-a-profile)
-    * [Load a profile](#load-a-profile)
-* [Non instrument data gathering](#non-instrument-data-gathering)
-  * [Log Files](#log-files)
-  * [Browser Profile](#browser-profile)
-  * [Rendered Page Source](#rendered-page-source)
-  * [Screenshots](#screenshots)
-  * [`save_content`](#save_content)
-
-# Platform Configuration Options
+- [Browser and Platform Configuration](#browser-and-platform-configuration)
+  - [Validations:](#validations)
+  - [Platform Configuration Options](#platform-configuration-options)
+  - [Browser Configuration Options](#browser-configuration-options)
+  - [Instruments](#instruments)
+    - [`http_instrument`](#http_instrument)
+    - [`js_instrument`](#js_instrument)
+    - [`navigation_instrument`](#navigation_instrument)
+    - [`callstack_instrument`](#callstack_instrument)
+    - [`dns_instrument`](#dns_instrument)
+    - [`cookie_instrument`](#cookie_instrument)
+  - [Browser Profile Support](#browser-profile-support)
+  - [Stateful vs Stateless crawls](#stateful-vs-stateless-crawls)
```

Comment:
Shouldn't `Stateful vs Stateless crawls` be a subsection of `Browser Profile Support`?